CHANGELOG.md

```diff
@@ -1800,10 +1802,12 @@ We may need to update the api documentation to reflect this.
 
 ## [v1.0.0] - 2018-02-23
 
-[Unreleased]: https://github.com/nasa/cumulus/compare/v1.16.1...HEAD
+[Unreleased]: https://github.com/nasa/cumulus/compare/v1.17.0...HEAD
+[v1.17.0]: https://github.com/nasa/cumulus/compare/v1.16.1...v1.17.0
 [v1.16.1]: https://github.com/nasa/cumulus/compare/v1.16.0...v1.16.1
 [v1.16.0]: https://github.com/nasa/cumulus/compare/v1.15.0...v1.16.0
-[v1.15.0]: https://github.com/nasa/cumulus/compare/v1.14.4...v1.15.0
+[v1.15.0]: https://github.com/nasa/cumulus/compare/v1.14.5...v1.15.0
+[v1.14.4]: https://github.com/nasa/cumulus/compare/v1.14.3...v1.14.5
```

Comment:
```suggestion
[v1.14.5]: https://github.com/nasa/cumulus/compare/v1.14.4...v1.14.5
```

Reply:
Can you also put the 1.14.5 changelog notes in here?

Reply:
Hmm, yes, I can.    I'm a little conflicted on doing this as a practice, though, as back port changes to previous releases aren't -technically- merged back to master or future releases. 

If we add the 1626 change to the log as version 1.14.5 in master, I think it may imply to users that  the change was made at that point and that future releases (e.g. 1.15.0) have it.

Reply:
> If we add the 1626 change to the log as version 1.14.5 in master, I think it may imply to users that the change was made at that point and that future releases (e.g. 1.15.0) have it.

I find this a very persuasive argument.

Reply:
Oh shoot I did it already. Well we put the comparison in there. And the date is in there as well.

Reply:
I think that's probably for the best. It occurred to me check and it looks like *none* of the backport docs are published to the github.io pages because they have not been merged back to master. 

I think getting backport versioned docs up is much more important than possible confusions about backport release dates, so I would support merging backports to master across the board.

Reply:
Doesn't look like this PR accomplishes merging those docs back though, but it's not necessarily in scope either. Something to think about.